pkg/rancher-components/src/components/Form/TextArea/TextAreaAutoGrow.vue

```diff
@@ -1,8 +1,16 @@
-<script>
+<script lang="ts">
+import Vue from 'vue';
 import debounce from 'lodash/debounce';
 import { _EDIT, _VIEW } from '@shell/config/query-params';
 
-export default {
+declare module 'vue/types/vue' {
+  /* eslint-disable no-unused-vars */
+  interface Vue {
+    queueResize(): void;
```

Comment:
Indeed, this would be much more simple if we added `queuesResize` to `data`, but I surmised that the original author had intent in adding `queueResize` as a non-reactive data property. I thought it was safer to augment the type in this case..

I recall a Vue github issue from a while back on this topic. I'm not sure if there might be something of use for us in there

https://github.com/vuejs/vue/issues/1988

Reply:
Thinking about this a little more, perhaps `queueResize` would make more sense as a method, like what we see in the debounce example from the vue docs

https://v2.vuejs.org/v2/guide/migration.html?redirect=true#debounce-Param-Attribute-for-v-model-removed